README.md

```diff
@@ -6,13 +6,18 @@
 ## Screenshot
 <img src='./screenshot.png'>
 
+## Presentation
+[Video](https://youtu.be/UfSsahg91uI)
+
 ## Getting started
   Run the following commands on the terminal your local machine:
 
   - `git clone https://github.com/Rayan84/catalog-of-things-ruby.git`
-  - `cd school-library`
+  - `cd catalog-of-things-ruby`
+  - `bundle`
   - `ruby the/path/to/main.rb`
```

Comment:
[OPTIONAL]

- In the README file add the command how to test so that other users would know how your projects should be tested. Since it is a small issue, I am approving your project but please add this before merging the PR.

Reply:
Well noted, thank you Dipesh